src/qt/utilitydialog.cpp

```diff
@@ -55,9 +55,9 @@ HelpMessageDialog::HelpMessageDialog(interfaces::Node& node, QWidget *parent, He
     {
         setWindowTitle(tr("About %1").arg(tr(PACKAGE_NAME)));
 
+        std::string licenseInfo = LicenseInfo();
```

Comment:
14521 should do https://github.com/bitcoin/bitcoin/pull/14521/files#diff-0adc60888360b5f3aa1e3cae614d4969854d74cbeebcd68a990e031a7529b834R26

Reply:
I checked this file and that include (which is in a different path in dash: https://github.com/dashpay/dash/blob/master/src/utilstrencodings.h ), this file doesn't use any function from utilstrencodings.h. I'll still include because it may be needed in a future backport. Just sharing what I've found.